dags/fetch_notices_for_date_range.py

```diff
@@ -1,13 +1,16 @@
 from datetime import datetime
 
+from airflow.decorators import dag, task
+from airflow.operators.python import get_current_context
 from airflow.operators.trigger_dagrun import TriggerDagRunOperator
 from dateutil import rrule
 
 from dags import DEFAULT_DAG_ARGUMENTS
-from airflow.decorators import dag, task
-from airflow.operators.python import get_current_context
-
 from dags.fetch_notices_per_day_worker import DATE_WILD_CARD_KEY
+from ted_sws.event_manager.adapters.event_log_decorator import event_log
+from ted_sws.event_manager.model.event_message import TechnicalEventMessage
+
+DAG_KEY = f"fetch_notices_for_date_range_{datetime.now().isoformat()}"
```

Comment:
This line executed once, I can't guarantee that this line will behave like DAG_RUN_ID (which is generated by airflow)  and is different at each DAG run.